src/classes.scss

```diff
@@ -44,3 +44,4 @@
 @import './wizard/classes';
 @import './skip-to-content/classes';
 @import './surface/classes';
+@import './radio-tile/classes';
```

Comment:
Same here with order.

Reply:
Changed order for radio tile, surface, and skip to content